### DISCUSSION TITLE
social share buttons broken a little

### DISCUSSION THREAD

**Discussion by duelmas** [Kovah/LinkAce#724] [Code Discussion]
So some of the social media share buttons I noticed had spaces instead of %20, etc...

**Comment by Kovah** [comment]
Hi, could you share a screenshot of the issue? The buttons look fine for me.

**Reply by duelmas** [reply]
![scrnla](https://github.com/Kovah/LinkAce/assets/143072270/80f50bd7-50a1-4522-b4c1-2e605e43356c)

**Reply by Kovah** [reply]
Ah yes, I thought you mean spacing in the layout. I will have a look, the URLs should be encoded.

**Reply by Kovah** [reply]
I double-checked this and there were no issues with the share links when using them. Do you run into issues when using this from a specific browser?

**Reply by duelmas** [reply]
I forget now it was such a while ago... Mainly noticing that the HTML does not validate because of the spaces are not %20.
I do remember the function when clicked on a few like that did not work but trivial minor issue kinda.... Still need a way to organize all the GBs of bookmarks for research purposes for multiple systems, operating systems and multiple users and browsers on each operating systems of thousands of bookmark.html bookmarks different folder organization needs... ready to export and import into a program that can automatically organize each link into appropriate easily understood categories, tags or labels.

I would like to share this GEM with you.... https://www.haztek-software.com/applications/storurl
Best software I have found so far for organizing massive databases of bookmark.html exports.... 
No docker needed, no local browser set-up, simple installation process. Light and runs locally. StorURL.
